app/views/widgets/shared/_footer.html

```diff
@@ -1,60 +1,61 @@
+            <li><a href="/students/tuition-and-aid/index">Tuition and Aid</a></li>
+            <li><a href="/campus-services/information-systems/policies-and-procedures/privacy-policy">Privacy</a></li>
           </ul>
         </div>
       </div>
       <div class="menu-section">
         <!-- SERVICES -->
         <div class="link-container">
-          <div class="links-header"><h3>Services</h3></div>
+          <div class="links-header"><h2>Services</h2></div>
           <ul class="link-list">
-            <li><a href="../../academics/libraries/index.aspx">Libraries</a></li>
-            <li><a href="../../students/health-and-safety/disability-services/index.aspx">Disability Services</a></li>
-            <li><a href="../../students/health-and-safety/title-ix.aspx">Title IX</a></li>
-            <li><a href="../../campus-services/public-safety/emergency.aspx">Emergency Information</a></li>
-            <li><a href="../../feedback-form.aspx">Feedback</a></li>
+            <li><a href="/academics/libraries/index">Libraries</a></li>
+            <li><a href="/students/health-and-safety/disability-services/index">Disability Services</a></li>
+            <li><a href="/students/health-and-safety/title-ix/index">Title IX</a></li>
+            <li><a href="/emergency/alert">Emergency Information</a></li>
+            <li><a href="/website-feedback-form">Feedback</a></li>
           </ul>
         </div>
         <!-- SOCIAL -->
         <div class="link-container">
-          <div class="links-header"><h3>Social</h3></div>
+          <div class="links-header"><h2>Social</h2></div>
           <ul class="link-list social-icon-list">
-            <li><a class="icon icon-twitter" href="http://twitter.com/#!/chapmanu" target="_blank"></a></li>
-            <li><a class="icon icon-facebook" href="http://www.facebook.com/ChapmanUniversity" target="_blank"></a></li>
-            <li><a class="icon icon-pinterest" href="http://pinterest.com/chapmanu/" target="_blank"></a></li>
-            <li><a class="icon icon-graduation" href="http://itunes.apple.com/us/institution/chapman-university/id430678922" target="_blank"></a></li>
-            <li><a class="icon icon-youtube" href="http://www.youtube.com/user/ChapmanUniversity" target="_blank"></a></li>
-            <li><a class="icon icon-linkedin2" href="http://www.linkedin.com/company/10701?trk=tyah" target="_blank"></a></li>
-            <li><a class="icon icon-google-plus" href="https://plus.google.com/112412382370481463024/posts" target="_blank"></a></li>
-            <li><a class="icon icon-instagram" href="http://www.instagram.com/chapmanu" target="_blank"></a></li>
+            <li><a class="icon icon-twitter" href="http://twitter.com/#!/chapmanu" target="_blank"><span class="sr-only">Twitter</span></a></li>
+            <li><a class="icon icon-facebook" href="http://www.facebook.com/ChapmanUniversity" target="_blank"><span class="sr-only"Facebook">Facebook</span></a></li>
```

Comment:
all the social icons should now have some text in the link but be surrounded by the span with sr-only class in order to hide it. I'm looking at the _footer.html that's part of the pull request and it looks right to me but I want to make sure I'm not misunderstanding something and there is actually an issue. The line should be the following, and I think it is. Can you look again and see if you still think it's wrong or I'm misunderstanding what you're reporting. thx:
<li><a class="icon icon-facebook" href="http://www.facebook.com/ChapmanUniversity" target="_blank"><span class="sr-only"Facebook">Facebook</span></a></li>